lib/plugins/features/movies/controller.js

```diff
@@ -8,6 +8,24 @@ exports.create = async (payload) => {
   Reflect.deleteProperty(payload, 'title');
 
   const movie = await new Movie().save(payload);
-
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.read = async (query) => {
```

Comment:
nit: name this method something like `findAll`, `getAll`, or `list` which might be more informative than `read`